ftw/datepicker/profiles/default_plone5/registry.xml

```diff
@@ -1,21 +1,13 @@
 <registry>
 
-    <record name="ftw.datepicker.interfaces.IDatetimeRegistry.formats">
-        <field type="plone.registry.field.Dict">
-            <title>The formats per language</title>
-            <key_type type="plone.registry.field.TextLine"/>
-            <value_type type="plone.registry.field.TextLine"/>
-        </field>
+    <record interface="ftw.datepicker.interfaces.IDatetimeRegistry" field="formats" name="ftw.datepicker.interfaces.IDatetimeRegistry.formats">
         <value purge="false">
             <element key="de">d.m.Y H:i</element>
             <element key="fr">d/m/Y H:i</element>
```

Comment:
Would you care to add a 'en' line too? 

 I'd like a copy of the 'fr' one but Americans might want m/d/Y ....   :-/

Reply:
en-gb vs en-gb

Reply:
probably the Yanks trump us for 'en' :-(

Reply:
Could you just add `<element key="en">d/m/Y H:i</element>`?  (it seems to fallback to "d.m.Y H:i" which isn't British OR American :-) )